include/Jit/LLILCJit.h

```diff
@@ -167,6 +186,10 @@ struct LLILCJitPerThreadState {
 /// top-level invocations of the jit is held in thread local storage.
 class LLILCJit : public ICorJitCompiler {
 public:
+  typedef llvm::orc::ObjectLinkingLayer<> LoadLayerT;
+  typedef llvm::orc::IRCompileLayer<LoadLayerT> CompileLayerT;
+  typedef CompileLayerT::ModuleSetHandleT ModuleHandleT;
```

Comment:
Nit: I assume you chose these names for consistency with ORC, but would you object to removing the terminal `T`?